hana/defaults.yaml

```diff
@@ -41,3 +41,8 @@ hana:
         remote_instance: '00'
         replication_mode: 'sync'
         operation_mode: 'logreplay'
+        scenario_type: 'cost-optimized'
```

Comment:
I wouldn't put these entries in the default.yml file. This is a quite specific deployment, so let's try to have it as generic as possible (,aybe we would need to remove some current values too)